common/changes/rush-update/change-auto-update-dep-job_2019-07-10-09-22.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "rush-update",
+      "comment": "Request pr review from michael when auto-updating dependencies",
```

Comment:
I think the change to the package of rush-update is "Introduced new flag --pr-reviewers", the reviewer in this case is part of configuration and not the change.